views/includes/page-templates/trust-legal-entity-bo.html

```diff
@@ -125,6 +125,8 @@ <h1 class="govuk-heading-xl govuk-!-margin-0">{{ pageParams.title }}</h1>
     {% set registration_number = formData.registration_number %}
     {% include "includes/inputs/fields/is-on-register-in-country-formed-input.html" %}
 
+    {% include "includes/inputs/fields/is-legal-entity-still-involved-in-trust.html" %}
```

Comment:
i've created a similar include but it is re-usable, once my PR is merged you can re-use that one as well.
something like this but set your own text values

`{% set ceasedDateLabel = "Date they ceased to be involved in the trust" %}
        {% set ceasedDateHint = "For example, 27 1 2023" %}
        {% set stillInvolvedLegendText = "Are they still involved in the trust?" %}
        {% include "includes/inputs/date/trust-ceased-date-input.html" %}`

Reply:
Changed templates to match this so that a merge will not change the functionality.

Reply:
LGTM.